gridscout

```diff
@@ -12,11 +12,16 @@ import sys
 from   schema import Schema, SchemaError
 import time
 
+import fcntl, termios, struct # Used in determining screen size
+
 class M:
-    log_filename  = 'log/gridscout.log'
-    key_filename  = 'config/google_places.key'
-    default_base  = '11T NJ 6337 2982'
-    prompt        = 'gridscout > '
+    log_filename = 'log/gridscout.log'
+    key_filename = 'config/google_places.key'
+    default_base = '11T NJ 6337 2982'
+    prompt       = 'gridscout > '
+
+    min_name_width = 25
+    other_width    = 50
```

Comment:
Set these two variables to tweak what gets truncated if you are displaying wide results on a narrow screen.